main/libpciaccess/ppc64le-limits.patch

```diff
@@ -0,0 +1,13 @@
+--- a/src/linux_sysfs.c
++++ b/src/linux_sysfs.c
+@@ -66,6 +66,10 @@
+ #include <sys/ioctl.h>
+ #endif
+ 
++#ifdef __PPC64__
++#include <limits.h>
```

Comment:
I think so. I just assumed it was not needed for other archs. Do you want me to test it here?